scripts/regridding/regrid_example.py

```diff
@@ -71,11 +71,16 @@ def regrid_example(case_name, input_climo_loc, output_loc,
             if not regridded_file_loc.is_file():
 
                 #Create list of regridding target files (we should explore intake as an alternative to having this kind of repeated code)
+                # NOTE: This breaks if you have files from different cases in same directory!
                 tclim_fils = sorted(list(tclimo_loc.glob("{}*_{}_*.nc".format(target, var))))
-
+                print(f"DEBUG: tclim_fils (n={len(tclim_fils)}): {tclim_fils}")
```

Comment:
Can you comment this print statement out and add an extra comment stating that it needs to be added to a debug log?